### ISSUE TITLE
Global Pesticide Grids (PEST-CHEMGRIDS), v1.01 

### ISSUE THREAD

**Issue by Mohinem** [FAIRiCUBE/data-requests#83]

**Comment by Mohinem** [comment]
Currently downloading the dataset, will come back with more questions during ingestion process.

**Comment by sonjastndl** [comment]
Cool Thanks! :)

**Comment by Mohinem** [comment]
@sonjastndl thank you for the update !

I need one more help. I noticed that the download wasn't complete because it's very slow (ranges from 10 - 100kbps). Can you help me find an alternative source to download ?

**Comment by Mohinem** [comment]
> Hi @Mohinem Unfortunately, this is the only provider of this data or the best estimator of pesticide use ... I will also have a look into what'S going on with the download. I am interested, what do you mean by slow? Do you mean the data is big or a slow download rate per time? How are you accessing the data?

Slow download rate. This should take a few minutes if the rate is good. 

![image](https://github.com/FAIRiCUBE/data-requests/assets/13696328/cce6318e-e3bb-4b91-bb20-cbe0ace0b35f)

**Comment by KathiSchleidt** [comment]
Hi @Mohinem strange, worked fine here. See if you can pull from here: https://www.dropbox.com/s/7ypcoan9b3oiom2/ferman-v1-pest-chemgrids-v1-01-geotiff.zip?dl=0

**Comment by sonjastndl** [comment]
@KathiSchleidt same for me, worked with the same when dowloading via your dropbox link

@Mohinem maybe you can try downlaoding via wget command ith option --tries==inf
, whenver I try to download big files at a slow rate

**Comment by Mohinem** [comment]
@KathiSchleidt your link helped the download, thank you.

I am going through the dataset, and have some questions for @sonjastndl .
There are three folders-

**Comment by sonjastndl** [comment]
@Mohinem Glad that worked out!

The three flolders should be a different layer each I would suggest. For the ApplicationRate Data: As far as I know there are several TIFFS that carry each information on: Crop, Pesticide, High/Low 
How feasable is it to summarize several Pestices in one layer? Can you summarize Information on e.g. Glyphosate - and then state information on Apllication Rate(kg/ha) for each crop?

Then we could ideally differnetiate between insecticides (chlorpyrifos,cyhalothrin,...) and herbicides (glyphosate etc). We could therfore analyse one active ingridient independet from crops it was applied for but also differentiate between them.
